pkgs/applications/networking/instant-messengers/chatterino2/default.nix

```
@@ -2,12 +2,12 @@
 
 mkDerivation rec {
   pname = "chatterino2";
-  version = "2.1.7";
+  version = "2.2.2";
   src = fetchFromGitHub {
     owner = "fourtf";
```

The repo is now hosted at https://github.com/Chatterino/chatterino2, so I think this should be updated to reflect that (as well as the `meta.homepage` below).

```suggestion
    owner = "Chatterino";
```